lib/screens/create_region_event_screens/add_region_event_name.dart

```diff
@@ -1,10 +1,50 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:seeds/components/flat_button_long.dart';
+import 'package:seeds/components/text_form_field_custom.dart';
+import 'package:seeds/domain-shared/ui_constants.dart';
+import 'package:seeds/screens/create_region_event_screens/interactor/viewmodels/create_region_event_bloc.dart';
 
 class AddRegionEventName extends StatelessWidget {
   const AddRegionEventName({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(appBar: AppBar(title: const Text("Create Event")));
+    return BlocBuilder<CreateRegionEventBloc, CreateRegionEventState>(builder: (context, state) {
+      return WillPopScope(
+          onWillPop: () async {
+            BlocProvider.of<CreateRegionEventBloc>(context).add(const OnBackPressed());
+            return false;
+          },
```

Comment:
Do you really need add the WillPopScope for each screen? Why not just in the main screen?

Reply:
I added it on the main screen and did not work.